admin/views/class-view-utils.php

```diff
@@ -108,7 +108,7 @@ public function show_post_type_settings( $post_type ) {
 			sprintf( __( '%1$s Meta Box', 'wordpress-seo' ), 'Yoast SEO' )
 		);
 
-		$editor = new WPSEO_Replacevar_Editor( $this->form, 'title-' . $post_type->name, 'metadesc-' . $post_type->name );
+		$editor = new WPSEO_Replacevar_Editor( $this->form, 'title-' . $post_type->name, 'metadesc-' . $post_type->name, false );
```

Comment:
For now I think it makes more clear to have a variable: `$paper_style = false` and passes that as the last argument. It's just an idea